app/src/main/res/drawable/heart_drawable.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@drawable/ic_big_heart"
+        android:state_pressed="true" />
+    <item android:drawable="@drawable/ic_big_heart" android:state_focused="true" />
+    <item android:drawable="@drawable/ic_big_heart" />
+</selector>
```

Comment:
not part of this PR but do you think any of these buttons should have pressed state icons?

I don't think we need the state_focused rules because we aren't changing the drawables.